libraries/clickhouse_base.rb

```diff
@@ -47,11 +47,7 @@ def action_delete
       end
 
       def rhel_family?
-        %w[rhel centos].include?(node['platform'])
-      end
-
-      def debian_family?
-        %w[debian ubuntu].include?(node['platform'])
+        %w[rhel centos rocky].include?(node['platform'])
```

Comment:
I think there is no such platform as `rhel`. Likely `redhat`: https://github.com/chef/chef/blob/61a11902ab814aad3625eb4da7e3345d63ee7c09/chef-utils/lib/chef-utils/dsl/platform.rb#L104